modules/ROOT/pages/index.adoc

```diff
@@ -4,6 +4,8 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: data gateway, salesforce, sap, oracle
 
+WARNING: Support for Anypoint Data Gateway continues in accordance with MuleSoft https://www.mulesoft.com/legal/support-maintenance-terms[Product Support and Maintenance Terms]. However, this product is not available for new customers or upgrades.
```

Comment:
This is just a general question and I'm assuming that since Kim edited it she is okay with using Warning instead of Important? Please ignore if it's already been addressed, just for my understanding.

Reply:
It's a good question. We're following the form used other support notifications, which is a "Warning." We'd need to think of whether we need to change that practice (perhaps globally).